fineract-provider/src/test/resources/application-test.properties

```diff
@@ -20,6 +20,8 @@
 fineract.node-id=1
 
 fineract.security.basicauth.enabled=true
+fineract.geolocation.enabled=false
+fineract.security.two-factor.enabled=false
```

Comment:
why the two-factor.enabled change?

Reply:
@JohnAlva Seems there is a bug here as we have `fineract.security.2fa.enabled=false` but not have `fineract.security.two-factor.enabled`, but in Fineract properties the field name is `twoFactor`

Reply:
I think we should raise an another PR to resolve this.